process.md

```diff
@@ -143,3 +143,9 @@ Thank you,
 The MicroProfile core team
 
 ---
+
+## Steps after a proposal has been accepted
+
+* Announce in the review mailing thread that the proposal has been accepted and that repositories are being created
+* Raise a bug for Eclipse to create a Github repsitory or ask for migration of an existing repository (see the [Eclipse Handbook](https://www.eclipse.org/projects/handbook/#resources-github))
+** When repositories are created, announce in a new thread that the proposal can be worked on in the new repository
```

Comment:
I think you're running into the problem I started to bring up in this thread: https://groups.google.com/forum/#!topic/microprofile/gtUUnntbPtk - basically we're going too deep in the proposals, which is causing us to think more of the implementation rather than the requirements.

I don't understand why that spec can't be merged either.  Was a vote had?